### PR TITLE
Moonlight Greatsword for Wizards

### PR DESCRIPTION
**Pull request by Weisyl**
Added the Moonlight Greatsword for wizards. To use it, it must be
charged from damaging non-crit humanoids 5 times. Once charged, you can
use it as a ranged weapon to fire 5 pellets dealing 15 brute each. After
firing, it must be fully charged once again. It can fit on your back.
All credits to FuryMcFury for the sprites.

http://puu.sh/mpPDy/897cf17e4b.png

http://puu.sh/mpPEu/91dbbdd4f0.png

http://puu.sh/mpPEU/3394b58195.png

http://puu.sh/mpPF7/090cdfe17c.png

### CODE REVIEW

File: html/changelogs/DerptheStewpidGoat - MoonlightSword.yml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+#   rscdel (general deleting of nice things)
+#   imageadd
+#   imagedel
+#   spellcheck (typo fixes)
+#   experiment
+#   tgs (TG-ported fixes?)
+#################################
+
+# Your name.  
+author: Dan Hibiki
```
Comment (TDcoolguy300) [inline html/changelogs/DerptheStewpidGoat - MoonlightSword.yml]: What

Comment (Weisyl) [inline html/changelogs/DerptheStewpidGoat - MoonlightSword.yml]: Oops

**Comment by trifornt-tinfornt** [comment]
Sounds potentially underpowered. You might as well just use Plasma Fist.

**Comment by danhibiki** [comment]
I'm for trying it, it bridges a gap between melee and ranged play for wizards

**Comment by FuryMcFlurry** [comment]
After reading the code, Is 15 brute dmg on a melee hit not a bit weak? after all you going to get targeted and what not. as for the 5 magic pellet dealing 15 dmg seems okey, if you can hit them with all 5 projectiles it will round up for 75 brute damage.

How much damage is acceptable? I dont want it to turn into an Esword for that matter. does 20-25 ish sound acceptable?

File: html/changelogs/DerptheStewpidGoat - MoonlightSword.yml (hunk 1 above)
Comment (danhibiki) [inline html/changelogs/DerptheStewpidGoat - MoonlightSword.yml]: i claim this for the house of hibiki

**Comment by calibraptor** [comment]
Sounds like a good weapon, so long as it's not incredibly expensive. You should be able to combo it with a couple spells. I'd recommend upping the pellet damage to 20 so actually getting in 5 hits is worth the destructive payout. I assume the pellets have spread like a shotgun.

**Comment by Nokington** [comment]
i'm up for trying it out

**Comment by Lawligagger** [comment]
I've actually been looking forward to this weapon.

**Pull request merged**